### ISSUE TITLE
Shop orders are sent to the person ordering not admin

### ISSUE THREAD

**Issue by Benleehy92** [pointsbot/bugs#12]
Hi there. When players in the server orders a store item, they are given a confirmation box saying "only visible to you", nothing is shared publicly or to admin, making us unaware that the coins have been spent or what order to process. Is there a fix planned for this? 
![IMG_2394](https://user-images.githubusercontent.com/121699928/210119723-89508bd6-5271-4e77-a42e-6f7d2669967c.png)

**Comment by GriffinSego** [comment]
Yep, this is definitely a bug. It should be easy to fix. Add me on VKontakte or Discord to show me a video of it?
